cheshire.mk

```diff
@@ -63,7 +63,7 @@ chs-clean-deps:
 ######################
 
 CHS_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:pulp-restricted/cheshire-nonfree.git
-CHS_NONFREE_COMMIT ?= 1e80dbc
+CHS_NONFREE_COMMIT ?= e244890 # yt/vcu118-rebase
```

Comment:
Since we only have one VCU118, we should make this a manually triggered task in nonfree CI, as we cannot guarantee it is available.

Reply:
Done